soroban-env-common/src/tuple.rs

```diff
@@ -32,15 +32,25 @@ macro_rules! impl_for_tuple {
             }
         }
 
-        impl<E: Env, $($typ),*> IntoVal<E, RawVal> for ($($typ,)*)
+        impl<E: Env, $($typ),*> IntoVal<E, Object> for ($($typ,)*)
```

Comment:
Can you also implement `TryFromVal<E, Object>`? The RawVal version is just above this function, and these implementations accompany each other.